integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
+sudo tar -xvf hbase-2.4.9-bin.tar.gz -C /opt/hbase --strip 1
+
+JAVA_HOME="$(dirname "$(dirname "$(readlink "$(readlink "$(which javac)")")")")"
+
+cat <<EOF | sudo tee /etc/systemd/system/hbase.service
+[Unit]
+Description=HBase Hadoop database
+After=network.target
+
+[Service]
+Type=forking
+Environment=JAVA_HOME=$JAVA_HOME
+ExecStart=/opt/hbase/bin/start-hbase.sh
+ExecStop=/opt/hbase/bin/stop-hbase.sh
+User=root
+Group=root
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+cat <<EOF | sudo tee /opt/hbase/conf/hbase-env.sh
+export HBASE_JMX_BASE="-Djava.rmi.server.hostname=0.0.0.0 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
```

Comment:
It's not important in this PR because this is just logs, but we'll need to think in the metrics PR about whether this is sufficient - I believe this only configures hbase in standalone mode, not distributed mode.

Reply:
Ok, we will address this on the metrics PR.